README.rst

```diff
@@ -24,17 +24,17 @@ About
 -----
 
 REANA-Client is a component of the `REANA <http://www.reana.io/>`_ reusable and
-reproducible research data analysis platform. It provides a command-line tool
```

Comment:
The word "analysis" should stay here and in other places that are using user-oriented language... You cannot "wide replace" everywhere.

Reply:
Right, though I didn't do a `replace all` actually as the word analysis here is just deleted.